spawn.go

```diff
@@ -27,7 +29,12 @@ func (s *Spawn) Run(store vaulted.Store) error {
 		ask.Print(fmt.Sprintf("%s — expires: %s (%s remaining)\n", session.Name, session.Expiration.Format("2 Jan 2006 15:04 MST"), timeRemaining))
 	}
 
-	code, err := session.Spawn(s.Command)
+	sshAgent, err := proxyagent.SetupAgent(proxyagent.AgentConfig{})
+	if err != nil {
+		return err
```

Comment:
I believe we need to wrap this `err` in an `ErrorWithExitCode{}`. See examples below.